src/components/GTFOBins/GtfoMain.jsx

```diff
@@ -0,0 +1,127 @@
+                                        {Object.keys(data[keys].functions).map((function_) => (
+                                            <button
+                                                onClick={() => {
+                                                    handleActiveFunction(function_);
+                                                }}
+                                                style={{ padding: "10px", border: "solid 1px", margin: "10px" }}
+                                                key={function_}
+                                            >
+                                                {function_}
+                                            </button>
+                                        ))}
+                                    </div>
+                                </div>
+                            );
+                        } else {
+                            if (Object.keys(data[keys].functions).find((func) => func === activeFunction)) {
+                                return (
+                                    <div style={{ display: "flex" }} key={keys}>
+                                        <a
+                                            style={{
+                                                width: "180px",
+                                                color: "white",
+                                                display: "flex",
+                                                alignItems: "center",
+                                                textDecorationLine: "underline",
+                                            }}
+                                            href={"/tools/binexploits/" + keys}
+                                        >
+                                            {keys}
+                                        </a>
+                                        <div style={{ display: "grid-row", width: "550px" }}>
+                                            {Object.keys(data[keys].functions).map((function_) => (
+                                                <button
+                                                    onClick={() => {
+                                                        handleActiveFunction(function_);
+                                                    }}
+                                                    style={{ padding: "10px", border: "solid 1px", margin: "10px" }}
+                                                    key={function_}
+                                                >
+                                                    {function_}
+                                                </button>
+                                            ))}
+                                        </div>
+                                    </div>
+                                );
+                            }
+                        }
+                        return null;
```

Comment:
if you `return null` so you should not use map() you can use forEach() instead , in my opinion